src/main/java/org/opensearch/performanceanalyzer/commons/collectors/ScheduledMetricCollectorsExecutor.java

```diff
@@ -159,4 +176,13 @@ public void run() {
             }
         }
     }
+
+    private boolean canSchedule(PerformanceAnalyzerMetricsCollector collector) {
+        if (collector instanceof TelemetryCollector) {
+            return (collectorsSetting == Util.CollectorMode.DUAL.getValue())
+                    || (collectorsSetting == Util.CollectorMode.OTEL.getValue());
```

Comment:
Can we please keep the names in sync like here OTEL should be replaced with telemetry or RTF.